Rationale.md

```diff
@@ -156,22 +156,29 @@ TODO
 
 ## Indirect Calls
 
-The exact semantics of indirect function calls, function pointers, and what
-happens when calling the wrong function, are still being discussed.
-
-Fundamentally linear memory is a simple collection of bytes, which means that
-some integral representation of function pointers must exist. It's desirable to
-hide the actual address of generated code from untrusted code because that would
-be an unfortunate information leak which could have negative security
-implications. Indirection is therefore desired.
```

Comment:
I'd like to keep this part of the rationale, or at least something that explains why we're trying to avoid using the function pointers as-is.